ygen/goelements_test.go

```diff
@@ -29,6 +29,7 @@ func TestFindChildren(t *testing.T) {
 	tests := []struct {
 		name             string
 		inElement        *yang.Entry
+		inExcludeState   bool
```

Comment:
We probably should use one terminology - i.e. ExcludeConfigFalse or ExcludeState.

Reply:
Agreed. Adopted 'state' throughout.